xsuite/src/world/world.ts

```diff
@@ -137,6 +138,12 @@ export class World {
     return this.proxy.getAccount(address);
   }
 
+  async getEsdtBalance(address: AddressLike, id: string, nonce?: number) {
```

Comment:
Let's also add `getAccountEsdtValue(address, id, nonce)` and use it in `getAccountEsdtBalance`. No need to add additional tests for this (as it will already be covered).